### ISSUE TITLE
Question: How to show the svg image in its native size?

### ISSUE THREAD

**Issue by BorzillaR** [microsoft/microsoft-ui-xaml#3830]
Hello!

I have a simple svg image

```
<svg
   xmlns="http://www.w3.org/2000/svg"
   width="20"
   height="20"
   version="1.1">

  <rect rx="0" ry="0" y="0" x="0" height="20" width="20" style="fill:red" />
</svg>
```
When I set the Image.Stretch property to None I expect the image size will be the same as svg size (20x20).

`<Image x:Name="image1" Source="MyImage.svg" Stretch="None" HorizontalAlignment="Center" erticalAlignment="Center"/>`

But actually the image ocupies the entire window area.

Setting Width="20" and Height="20" solve this. But in my scenario I do not know the svg image size and I dont want to parse the svg file to get the size.

**Comment by StephenLPeters** [comment]
@BorzillaR what are the layout properties of the image? I think Horizontal alignment = stretch could cause this behavior as well.

**Comment by BorzillaR** [comment]
@StephenLPeters, you are right. 

I tried to set different values for Image.HorizontalAlignment/VerticalAlignment/Stretch properties but without success. The image still has a large size.

For more clarity, I create an image with the one red rectangle inside and put it into the green border.

*The svg image*
![image](https://user-images.githubusercontent.com/23479280/102751996-ea8ac980-4379-11eb-9367-bec1f7c65529.png)

*Xaml*
![image](https://user-images.githubusercontent.com/23479280/102752270-5cfba980-437a-11eb-8a33-99042a908242.png)

Then I set the different properti values and see the results.

None of my experiments ended with success.

I also noticed that the svg content is drawn in size 20x20 as I want, but the image is still large.

![image](https://user-images.githubusercontent.com/23479280/102752570-d09db680-437a-11eb-8f99-21f3bc287b04.png)
